webdriver-spec.html

```diff
@@ -2018,10 +2022,12 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
 </table>
 
-<p>A <dfn>host and optional port</dfn> is defined as being a
-  valid <a>host</a>, optionally followed by a colon
-  and <a>port</a>. The <a>host</a> may
-  <a data-lt="includes credentials">include credentials</a>.
+<p>A <dfn>host and optional port</dfn> for a <var>scheme</var> is
```

Comment:
I really think we can drop this whole definition because it is a redefinition really of [URL](https://url.spec.whatwg.org/#concept-url). This means that we can, for the other parts, instead of
`<td>A <a>host and optional port</a> with an <a>undefined</a> scheme.` 
we can say 
`<td>A <a>URL</a> with an <a>undefined</a> scheme.`

Reply:
But it's not a complete URL, since we omit the scheme in valid values for a proxy. In order to specify the default port, we then need to perform linguistic shenanigans like saying `a URL with an undefined scheme but with port set to the default port for "scheme" if the port is omitted`, which ain't elegant at all.